docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
+- When choosing the partitioning granularity, you need to consider factors data volume, query patterns, and data management granularity.
+  - Example 1: If the monthly data volume in a table is small, partitioning by month can reduce the amount of metadata compared to partitioning by day, thereby reducing the resource consumption of metadata management and scheduling.
+  - Example 2: If the monthly data volume in a table is large and queries mostly request data of certain days, partitioning by day can effectively reduce the amount of data scanned during queries.
+  - Example 3: If the data needs to expire on a daily basis, partitioning by day is recommended.
+
+#### Bucketing
+
+The bucketing method divides a partition into multiple buckets. Data in a bucket is referred to as a tablet.
+
+The supported bucketing methods are [random bucketing](#random-bucketing-since-v31) (from v3.1) and [hash bucketing](#hash-bucketing).
+
+- Random bucketing: When creating a table or adding partitions, you do not need to set a bucketing key. Data within a partition is randomly distributed into different buckets.
+
+- Hash Bucketing: When creating a table or adding partitions, you need to specify a bucketing key. Data within the same partition is divided into buckets based on the values of the bucketing key, and rows with the same value in the bucketing key are distributed to the corresponding and unique bucket.
+
+The number of buckets: By default, StarRocks automatically sets the number of buckets (from v2.5.7). And you can also manually set the number of buckets. For more information, please refer to [determining the number of buckets](#determine-the-number-of-buckets).
+
+## Create and manage partitions
 
-- The second level is bucketing: within a partition, data in a partition needs to be further distributed into smaller bucktes. There are two ways to distribute data into buckets, Hash and Random distribution. As for the  number of buckets, StarRocks can automatically set the number of buckets (recommended) or you can specify it manually.
+### Create partitions
 
-> **NOTE**
+#### Expression partitioning (recommended)
+
+> **NOTICE**
 >
-In addition to common distribution methods, StarRocks also supports Random distribution, which simplies bucketing configuration.
+> Since v3.1,  StarRocks's [shared-data mode](../deployment/deploy_shared_data.md) supports the time function expression and does not support the column expression.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/69da9cf5-96d7-4543-8233-f4b8038ef3ed)
![image](https://github.com/StarRocks/starrocks/assets/57167462/826df79a-4190-4ebf-b1ce-0deb5cef4c2a)
The Chinese version doesn't include the "does not support the column expression" part. Pls check.